dashboard/config/scripts_json/coursee-2021.script_json

```diff
@@ -7480,7 +7480,7 @@
       "seeding_key": {
         "level.key": "courseE_farmer_while1_2021",
         "script_level.level_keys": [
-          "courseE_farmer_ramp12g_2021"
+          "courseE_farmer_while1_2021"
```

Comment:
it is clear this line needs to be updated, because script_level.level_keys does not match level.key.